### PR TITLE
Upgrading modules in Aijar

### PR DESCRIPTION
**Pull request by slubwama**
Upgrade Platform and Modules in Aijar
https://app.asana.com/0/1133167201254913/1163190653310703

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -89,17 +89,17 @@
 		<providermanagementVersion>2.10.0</providermanagementVersion>
 		<referenceapplicationVersion>2.10.0-SNAPSHOT</referenceapplicationVersion>
 		<referencemetadataVersion>2.10.1</referencemetadataVersion>
-		<registrationappVersion>1.15.0-SNAPSHOT</registrationappVersion>
+		<registrationappVersion>1.15.0</registrationappVersion>
 		<registrationcoreVersion>1.9.0</registrationcoreVersion>
 		<reportingRestVersion>1.11.0</reportingRestVersion>
 		<reportingVersion>1.20.0-SNAPSHOT</reportingVersion>
```
Comment (ssmusoke) [inline pom.xml]: @slubwama Reporting is still a snapshot

File: pom.xml [hunk 2]
```diff
@@ -89,17 +89,17 @@
...
 		<reportingCompatibilityVersion>2.0.6</reportingCompatibilityVersion>
 		<reportingUIVersion>1.6.0</reportingUIVersion>
 		<serializationxstreamVersion>0.2.14</serializationxstreamVersion>
-		<uicommonsVersion>2.11.0-SNAPSHOT</uicommonsVersion>
+		<uicommonsVersion>2.11.0</uicommonsVersion>
 		<uiframeworkVersion>3.16.0</uiframeworkVersion>
 		<uilibraryVersion>2.0.6</uilibraryVersion>
-		<ugandaemrfingerprintVersion>1.0.10</ugandaemrfingerprintVersion>
+		<ugandaemrfingerprintVersion>1.0.11-SNAPSHOT</ugandaemrfingerprintVersion>
```
Comment (ssmusoke) [inline pom.xml]: @slubwama What is in this snapshot? Do I need to release?

Comment (slubwama) [reply pom.xml]: This Pr is still pending https://github.com/METS-Programme/openmrs-module-ugandaemr-fingerprint/pull/38

File: pom.xml (hunk 1 above)
Comment (slubwama) [reply pom.xml]: Yes it is

**Comment by ssmusoke** [comment]
@slubwama can you also bump up the OpenMRS Platform version to 2.3.0 it has the required fix

**Comment by slubwama** [comment]
@ssmusoke we cant yet. It breaks every thing. all tests fail. there is a-lot to do. can you get 2.2.1  released.

**Comment by sherrif10** [comment]
Will start volunteering in this community, the problem is that issues raised must be assigned to the people involved, inotherwords i cant be able find the tickets to work on , wanted to be working behind scenes

**Comment by sherrif10** [comment]
cc @ssmusoke

**Comment by slubwama** [comment]
@dkayiwa Need your help on upgrading core to 2.3.0. The tests in org.openmrs.module.aijar.SettingsNotificationTest are failing and I see the cause is related to `An association from the table allergy_reaction refers to an unmapped class: org.openmrs.Allergy` Also I see there is a change in the allergy_reaction table where column comment is changing to comments. This may have not been changed in the base test data to reflect the new changes. Just need clarity and correction on this since this test is failing due to this.

**Comment by dkayiwa** [comment]
@slubwama is this what you are looking for? https://talk.openmrs.org/t/problem-running-tests-against-openmrs-core-2-3-0/25498

**Pull request merged**